### ISSUE TITLE
[coinex] fetch_deposits bug

### ISSUE THREAD

**Issue by ethanbrown3** [ccxt/ccxt#6732]
When parsing a transaction after fetching deposits there is a bug with calling `parse_transaction_status` method. It is passing and extra argument in of `type` on line 685. I've been able to make this change on that line in my local site package and it works as expected: 
```[python]
status = self.parse_transaction_status(self.safe_string(transaction, 'status'))
```
I'd be happy to submit a pull request.

- OS: MacOS 10.15.3 but running in a docker container `python3.7`
- Programming Language version: 3.7.4
- CCXT version:  1.25.67 

```
>>> coinex = ccxt.coinex(config=config)
>>> deposits = coinex.fetch_deposits(code='ETH')
```

```
Traceback (most recent call last):
  File "<input>", line 1, in <module>
    coinex.fetch_deposits(code='ETH')
  File "/usr/local/lib/python3.7/site-packages/ccxt/coinex.py", line 805, in fetch_deposits
    return self.parse_transactions(response['data'], currency, since, limit)
  File "/usr/local/lib/python3.7/site-packages/ccxt/base/exchange.py", line 1619, in parse_transactions
    array = [self.extend(self.parse_transaction(transaction, currency), params) for transaction in array]
  File "/usr/local/lib/python3.7/site-packages/ccxt/base/exchange.py", line 1619, in <listcomp>
    array = [self.extend(self.parse_transaction(transaction, currency), params) for transaction in array]
  File "/usr/local/lib/python3.7/site-packages/ccxt/coinex.py", line 685, in parse_transaction
    status = self.parse_transaction_status(self.safe_string(transaction, 'status'), type)
TypeError: parse_transaction_status() takes 2 positional arguments but 3 were given
```

**Comment by kroitor** [comment]
Thank you so much for your report! Fixed it in 1.25. The new build will arrive in 15 minutes.

> I'd be happy to submit a pull request.

Your contributions are welcome and we really appreciate all help! If you have a one-liner fix like this, you can simply submit the PR directly next time, because a PR is also an issue ) However, please make sure to read the following: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#multilanguage-support before submitting. The changes have to be added to the source js file, but it's not that hard, since the js←→python code is transpiled line-to-line, so any developer can understand it and modify it, even without having to know the other languages.

Thx again for your involvement!

**Comment by ethanbrown3** [comment]
sounds good! thanks! I wasn't sure the process. I'll read that doc and on the next thing I'll find I'll submit the PR.
